_posts/2017-05-18-install-party.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: "Install Party + Social"
+slug: install-party
+text: 'tomesh monthly install party and social'
+location: Semaphore Studio 307, Claude T. Bissell, 140 St. George St  
+locationLink: http://osm.org/go/ZX6Bw~WNh--?m=
+date: 2017-05-18
```

Comment:
Right! We agreed on a monthly install party and a biweekly hardware (node.tech) meeting at the last planning session-
I understand the one on the 4th is more of a hardware meeting, so should I add it as such? I figure the calendar would end up looking something like:
- May 04 - node.tech
- May 18 - install party/check-in
- Jun 01 - node.tech
- Jun 15 - node.tech
- Jun 29 - install party/check-in
Thoughts?